style.css

```diff
@@ -0,0 +1,658 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+}
+
+.blue {
+  color: #6070ff;
+}
+
+.head {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  display: flex;
+  min-height: 10vh;
+  padding: 10px 20px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header.head {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #636fd4;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.toolbar > a {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
+  text-decoration: none;
+  padding: 0 24px;
+}
+
+.link a {
+  width: 15px;
+  margin-top: 10px;
+  margin-right: 10px;
+}
+
+.popup a {
+  padding: 8px 32px;
+  text-decoration: none;
+  font-size: 25px;
+  color: #fff;
+  display: block;
+  transition: 0.3s;
+  margin-right: 8px;
+}
+
+header.head a {
+  text-decoration: none;
+  color: #6070ff;
+}
+
+main {
+  margin: 0 auto;
+  height: 40px;
+  padding: 0 27px;
+}
+
+.bruw {
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.menu {
+  padding: 3px 5px;
+}
+
+.navigation_option {
+  display: none;
+  list-style: none;
+  column-gap: 20px;
+}
+
+#toppart {
+  display: grid;
+  row-gap: 60px;
+  background-color: #fff;
+}
+
+.toolbar h1 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  color: #253652;
+  padding: 0 20px;
+  margin: 0;
+}
+
+#toppart h1 {
+  font-style: normal;
+  font-weight: 800;
+  font-size: 50px;
+  color: #253652;
+  padding: 0 24px;
+}
+
+.toolbar {
+  width: 100%;
+  height: calc(100vh - 48px);
+  background-size: 100% 100%;
+  padding-top: 114px;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.start {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  margin-top: 60px;
+  margin-bottom: 20px;
+  font-weight: 700;
+  color: #172b4d;
+  line-height: 52px;
+  position: relative;
+}
+
+.mid {
+  margin: 10px 0;
+  color: #344563;
+  font-weight: 400;
+  background: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  line-height: 30px;
+  font-size: 16px;
+  font-size-adjust: 5;
+}
+
+.middle {
+  padding: 100px;
+  background: white;
+  margin-right: 100px;
+  display: column;
+}
+
+.blu {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: normal;
+  color: #6070ff;
+}
+
+.socials {
+  display: flex;
+  gap: 5px;
+  list-style: none;
+  padding: 0 24px;
+}
+
+.astaricks {
+  max-width: 100%;
+}
+
+.contruct {
+  margin: 25px 0;
+  display: grid;
+  row-gap: 25px;
+  padding: 10px;
+  background: #e5e5e5;
+  justify-items: center;
+}
+

```

Comment:
- [ ] as the previous CR mentioned there is still a vertical scroll in your project, you can try to fix this by adding a max-width for your sections and giving the value of 100vw, this means it will always be the same width as your screen not larger.